Formula/gnuradio.rb

```diff
@@ -72,18 +65,11 @@ class Gnuradio < Formula
     sha256 "964031c0944f913933f55ad1610938105a6657a69d1ac5a6dd50e16a679104d5"
   end
 
-  # patch for boost 1.73.0, remove after next release
-  # https://github.com/gnuradio/gnuradio/pull/3566
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/0d2af1812716a874d1e49268e999ea1a8ca9fc3c/gnuradio/boost-1.73.0.patch"
-    sha256 "7e4abd08210d242d65807b7e2419f163a58b4630027a3beaff0e325d044266d7"
-  end
-
-  # Add -undefined dynamic_lookup linker flag back for macOS
-  # https://github.com/gnuradio/gnuradio/pull/3674
+  # patch to fix drag-and-drop in gnuradio-companion
+  # https://github.com/gnuradio/gnuradio/issues/2727
```

Comment:
Could you please add a note that this can be deleted in the next release.
Or only https://github.com/gnuradio/gnuradio/pull/3980 is not enough to fix this?